.github/workflows/ci.yml

```diff
@@ -143,7 +143,8 @@ jobs:
         run: rush test -o chat
 
   build_static_html_composites_sample:
-    name: Build Static HTML Composites Sample
+    if: ${{ github.ref == 'refs/heads/main' || github.event_name == 'pull_request' }}
```

Comment:
The CI only runs on `push: [main]` and `pull_request` - is this if scoping this further?

Reply:
I believe I can remove this one, this is not needed anymore